electrumsv/tests/data/wallets/26_regtest_standard_mining_with_mature_and_immature_coins_history_full.json

```diff
@@ -86,27 +108,5 @@
         "tx_id": "b50efbc7b29d1943ca54f90213845c9bf29543f176be52df73d4eefcdd658c53",
         "block_id": null,
         "script_pubkey_hex": "76a91411e4d512620a42f4c3e376dfaf695bd11d71011488ac"
-    },
-    {
-        "txo_index": 0,
```

Comment:
What does it mean for these two to be removed?

Reply:
It was really just based on the fact that `read_history_for_outputs` has a default `limit=10` kwarg... So by adding these two new txs to the top 10 list, it bumps these other two transactions out of the top 10.

I am not aware of there being any other significance to these removed txs than that.